server/src/services/sendEmail.js

```diff
@@ -36,28 +36,36 @@ const datastore= new Datastore({
 //   });
 // };
 
-export const sendEmail = async ({email, message}) => {
+export const sendEmail = async ({sendRequest}) => {
   
   const envi = process.env.NODE_ENV || 'development';
-  if (envi=='development') {
-    return 'le feedback a été envoyé(une reponse automatique en mode '+
-      'de developement)';
-  }
-  
-  const template = replaceHtmlVars(emailTemplate , message , email);
-  const msgTemp =   {
-    to: email,
+  const template = replaceHtmlVars(emailTemplate , sendRequest);
+
+  const msgDev =   {
+    to: 'bnyat.azizsharif@zenika.com',
     from: 'bnyat.azizsharif@zenika.com',
     subject:'FeedZback',
     html: template,
   }
+  const msgProd =   {
+    to: sendRequest.receverEmail,
+    from: sendRequest.email,
+    subject:'FeedZback',
+    html: template,
+  }
```

Comment:
Ici tu construit deux objets qui ne sont pas utilisé tous les deux ...
```suggestion
const msg =   {
    to: sendRequest.receverEmail,
    from: sendRequest.email,
    subject:'FeedZback',
    html: template,
  }
  if (envi === 'developpement') {
    msg = {
      ...msg,
      to: 'bnyat.azizsharif@zenika.com',
      from: 'bnyat.azizsharif@zenika.com',
    };
  }
  
```
Tu peux avoir un code de ce style.

Reply:
stylé! merci